tests/integration_tests.rs

```diff
@@ -1225,3 +1225,25 @@ fn grid_for_file_without_newline() {
         )
         .stderr("");
 }
+
+#[test]
+fn ignored_suffix_arg() {
+    bat()
+        .arg("-f")
+        .arg("-p")
+        .arg("test.json~")
+        .assert()
+        .success()
+        .stdout("\u{1b}[38;5;231m{\u{1b}[0m\u{1b}[38;5;208m\"\u{1b}[0m\u{1b}[38;5;208mtest\u{1b}[0m\u{1b}[38;5;208m\"\u{1b}[0m\u{1b}[38;5;231m:\u{1b}[0m\u{1b}[38;5;231m \u{1b}[0m\u{1b}[38;5;186m\"\u{1b}[0m\u{1b}[38;5;186mvalue\u{1b}[0m\u{1b}[38;5;186m\"\u{1b}[0m\u{1b}[38;5;231m}\u{1b}[0m")
+        .stderr("");
+
+    bat()
```

Comment:
Thanks a lot for adding tests! Tests are very valuable. Almost more valuable than the code they test 😉

It would be good to also test that `test.json.suffix` behaves as expected when `--ignored-suffix` is NOT used. This will reduce the risk of the test passing even if `--ignored-suffix` breaks. It's an easy way to increase robustness of the test.

Reply:
Thanks a lot for extending the test, it looks even better now 👍